docs/docs/adding-an-rss-feed.md

```diff
@@ -18,7 +18,7 @@ npm install --save gatsby-plugin-feed
 
 ## How to use [gatsby-plugin-feed](/packages/gatsby-plugin-feed/)
 
-Once installation is complete, you can now add this plugin to your site's config file, like so:
+Once you complete installation, you can add the plugin to your site's config file, like so:
```

Comment:
This is an npm installation? So it'd be "once NPM finishes installation"

Reply:
Good question! I edited for passive voice, not for content, but can definitely change that portion!

Reply:
Edited to make it even more concise:

> At this point, you can add the plugin to your site's config file, like so: